data/easy_chat.s

```diff
@@ -11,13 +11,42 @@ gUnknown_08597550:: @ 8597550
 	.incbin "baserom.gba", 0x597550, 0x1f8
 
 gUnknown_08597748:: @ 8597748
-	.incbin "baserom.gba", 0x597748, 0x1c
+	.byte 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x00, 0x07, 0x08, 0x09, 0x0a, 0x0b, 0x0c, 0x00, 0x0d, 0x0e, 0x0f, 0x10, 0x11, 0x12, 0x13, 0x14, 0x15, 0x16, 0x17, 0x18, 0x19, 0x1a
 
 gUnknown_08597764:: @ 8597764
-	.incbin "baserom.gba", 0x597764, 0x8
+	.2byte 0x0209, 0x140b, 0x1030, 0x102a
 
 gUnknown_0859776C:: @ 859776C
```

Comment:
597780 is a palette, and 5977A0 is the start of some graphics data; you should extract this.